readme.md

```diff
@@ -0,0 +1,46 @@
+# pycytominer-transform
+
+```mermaid
+    flowchart LR
+
+        images[(Images)]
+        CellProfiler
+        csv[(CSV)]
+        DeepProfiler
+        npz[(NPZ)]
+        cytominer-database
+        sqlite[(SQLite)]
+        cp_sqlite[(SQLite)]
+        pycytominer
+        pycytominer-transform
+        parquet[(Parquet)]
+
+        images --> CellProfiler
+        images --> DeepProfiler
+        CellProfiler --> csv
+        CellProfiler --> cp_sqlite
+        DeepProfiler --> npz
+        csv --> cytominer-database
+        cytominer-database --> sqlite
+        csv --> pycytominer-transform
+        npz --> pycytominer-transform
+        sqlite --> pycytominer-transform
+        cp_sqlite --> pycytominer-transform
+        pycytominer-transform --> parquet
+        parquet --> pycytominer
+```
+
+_Diagram showing data flow relative to this project._
+
+## Summary
+
+Pycytominer Transform will take inputs in the form of CSV's, SQLite, or NPZ, converting to [Pycytominer](https://github.com/cytomining/pycytominer) relevant output in parquet format.
+The parquet files will have a unified and documented data model, including referenceable schema where appropriate (for validation within Pycytominer or other projects).
+
+## References
```

Comment:
I don't know if this README is just a stub at the moment, so no worries if the stuff I'm suggesting is coming. That said, I'd like to see some extra sections here: one on "Requirements" (e.g., poetry), one on "Installation" (e.g., how to use poetry to set up the project), one on "Usage" (potentially pointing to your tutorial), and one on "Testing" (e.g., where the test suite is, what prereqs if any are required to run it, and what command to run). 

Since I assume this is ultimately going to be a package on PyPI, you might consider making "Requirements", "Installation", and "Testing" subsections of a "Developing pycytominer-transform" section.

Reply:
This is definitely a gap, thank you for highlighting! I'll add these to the readme here in a later commit so as to address your comments.

Reply:
In working through this comment I thought it would be most helpful to begin creating `contributing` content. I moved `development.md` to become `contributing.md` (with development as a sub-section). `readme.md` now references this content and also calls out installation separately. The installation procedure included is a placeholder until the package is released to PyPI. The content within contributing is also just a start and will eventually contain more than just development content.

Note: in the process of doing this I also sought to reduce content duplication, making the root `readme.md` become the majority of the content on sphinxdocs index.md via myst's [{include}](https://myst-parser.readthedocs.io/en/latest/faq/index.html?#include-a-file-from-outside-the-docs-folder-like-readme-md).

Reply:
Thanks @d33bs - I agree a contributing doc would be great. Just stopping by to point to pycytominer's contributing document: https://github.com/cytomining/pycytominer/blob/master/CONTRIBUTING.md

Feel free to copy some of the words/sentiment

Reply:
Thank you @gwaybio ! I feel like this content would be a great addition and will add it in with some modifications where appropriate.